src/test/rustdoc-ui/coverage/output-format-fail-check.rs

```diff
@@ -0,0 +1,4 @@
+// compile-flags:-Z unstable-options --output-format
```

Comment:
What was this supposed to test? Was there meant to be something passed to `--output-format`? As this test is in the `coverage` directory, was `--show-coverage` supposed to be passed here?

Reply:
It's checking that it fails when no parameter is provided.

Reply:
In that case it shouldn't be in the `coverage` directory and `-Z unstable-options` doesn't need to be passed. However more importantly, the error this test is actually producing is `error: too many file operands` because compiletest is adding more arguments after `--output-format` which are then getting misinterpreted by rustdoc. It may be best to just drop this test.